src/NewTools-Playground/StPlaygroundBindingModif.class.st

```diff
@@ -0,0 +1,59 @@
+Class {
+	#name : 'StPlaygroundBindingModif',
+	#superclass : 'StPlaygroundBindingPresenter',
+	#instVars : [
+		'oldBinding'
+	],
+	#category : 'NewTools-Playground-View',
+	#package : 'NewTools-Playground',
+	#tag : 'View'
+}
+
+{ #category : 'accessing' }
+StPlaygroundBindingModif class >> model: anModelPresenter [
+	^ self basicNew 
```

Comment:
Can't we call #on: and set all of this in #setModelBeforeInitialization: since all parameters are set using anModelPresenter.

Also, aModelPresenter and not anModelPresenter